src/main/java/hudson/tasks/Mailer.java

```diff
@@ -480,11 +480,7 @@ public String getSmtpServer() {
          * @throws IllegalStateException if the object is not available (e.g., Jenkins not fully initialized).
          */
         private JenkinsLocationConfiguration getJenkinsLocationConfiguration() {
```

Comment:
Would better be inlined now, as the purpose as stated in Javadoc is apparently obsolete.